### DISCUSSION TITLE
Tone Mapping strings translation

### DISCUSSION THREAD

**Discussion by kajaji** [Stellarium/stellarium#2225] [Translations]
Where can the strings of tone mapping be translated? I couldn't find them in "stellarium" translation strings.

![toneMapping](https://user-images.githubusercontent.com/34771982/151808458-0684edf4-ba61-4883-8760-2461ce4d3387.png)

**Comment by alex-w** [comment]
Oops… my fault - I forgot adding these strings into list of translatable data
